src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -269,4 +271,21 @@ private function getAvailableFormatOptions(): array
     {
         return ['txt', 'yaml', 'json'];
     }
+
+    private function getDocumentationUrl(ExtensionInterface $extension, ContainerBuilder $container): ?string
+    {
+        $configuration = $extension instanceof ConfigurationInterface ?
+            $extension :
+            $extension->getConfiguration(
```

Comment:
Our CS is the following (and we usually prefer oneliners) :
```suggestion
        $configuration = $extension instanceof ConfigurationInterface
            ? $extension
            : $extension->getConfiguration(
```

Reply:
Indeed! Updated with one-liners